lib/diaspora_federation/entities/related_entity.rb

```diff
@@ -24,6 +24,14 @@ class RelatedEntity < Entity
       #   @return [RelatedEntity] parent entity
       entity :parent, Entities::RelatedEntity, default: nil
 
+      # The most top entity, this entity is responsible for relaying relayables
+      # @return [RelatedEntity] absolute parent entity
+      def top_entity
```

Comment:
Not sure "Top entity" is the best wording here, maybe something like "relayable responsible"? I don't know, but something containing the meaning of what we're doing that here.

Reply:
I'm not sure too. `top_entity` is short and says what it is. The documentation above the method definition describes for what it's used with more words. Before `parent` was used instead of `relayable_responsible`, even when it was what it was doing before.

Reply:
`root_entity` maybe?

Reply:
I also thought about `root_...` but I thought it could be confusing with `root_...` at reshares :/

Reply:
Yeah, I thought so as well. However, `root` is actually a pretty accurate word, since `root` is always the top level. Eventually, this will also be true for reshares, so maybe, reusing the same terminology is a good way to go?

Reply:
I renamed it to `root`. While it's maybe a bit weird now, because we have two things named `root`, I think since reshares won't have comments and likes (relayables) in the future, this will not create conflicts there in the future.